nodes/cma/nodes-cma-autoscaling-custom.adoc

```diff
@@ -49,3 +49,21 @@ Successfully updated ScaleTarget
 ----
 
 You can temporarily pause the autoscaling of a workload object, if needed. For example, you could pause autoscaling before performing cluster maintenance.
+
+.Custom metrics autoscaler workflow
+image::564_OpenShift_Custom_Metrics_Autoscaler_0224.png[Custom metrics autoscaler workflow]
+
```

Comment:
Up above in line 19 the text says "you" and below in line 56 the text says "A user." Unless there is a reason otherwise, I would try to use a similar structure and constant phrasing. So for line 56 maybe something like: "You create or modify a scaled object custom resource for a workload on the cluster."

Reply:
Agree that the references should be made consistent.